sylvia-derive/src/message.rs

```diff
@@ -1662,65 +1624,61 @@ impl<'a> EntryPoints<'a> {
             None => quote! {},
         };
 
-        #[cfg(not(tarpaulin_include))]
-        {
-            let entry_points = [instantiate_variants, exec_variants, query_variants]
-                .into_iter()
-                .map(
-                    |variants| match get_entry_point(override_entry_points, variants.msg_ty) {
-                        Some(_) => quote! {},
-                        None => variants.emit_default_entry_point(
-                            &custom_msg,
-                            &custom_query,
-                            name,
-                            error,
-                            &attrs.generics,
-                        ),
-                    },
-                );
-
-            let migrate_not_overridden =
-                get_entry_point(override_entry_points, MsgType::Migrate).is_none();
-
-            let migrate = if migrate_not_overridden && migrate_variants.get_only_variant().is_some()
-            {
-                migrate_variants.emit_default_entry_point(
-                    &custom_msg,
-                    &custom_query,
-                    name,
-                    error,
-                    &attrs.generics,
-                )
-            } else {
-                quote! {}
-            };
+        let entry_points = [instantiate_variants, exec_variants, query_variants]
+            .into_iter()
+            .map(
+                |variants| match get_entry_point(override_entry_points, variants.msg_ty) {
+                    Some(_) => quote! {},
+                    None => variants.emit_default_entry_point(
+                        &custom_msg,
+                        &custom_query,
+                        name,
+                        error,
+                        &attrs.generics,
+                    ),
+                },
+            );
```

Comment:
We might keep disabling tarpaulin here. Unfortunately it seems that it still has some troubles with coverage inside the iterators.

Reply:
As we discussed offline, I'd keep this part as not test-covered so we could research for the solution for the problem in the future.